documentation/asciidoc/topics/proc_configuring_sites_automatically.adoc

```diff
@@ -4,6 +4,16 @@
 [role="_abstract"]
 Configure {ispn_operator} to establish cross-site views with {brandname} clusters.
 
+[NOTE]
+====
+Cross-site replication does not support the {k8s} CA service, so you must choose one of the following options:
+
+* Create a certificate and then add it to each client that connects to a {brandname} endpoint.
```

Comment:
If we add a Note to `documentation/asciidoc/topics/proc_securing_cross_site_connections.adoc`, then I think it's oc to simply state something like:

"Cross-site replication does not support the {k8s} CA service, therefore it's necessary to provide your own certificates."